kurl_proxy/assets/tls-custom.css

```diff
@@ -399,6 +399,7 @@ button.text-button:hover {
   overflow: hidden;
   position: absolute;
   z-index: -1;
+  opacity: 0;
```

Comment:
hide the default upload putting. this was manually aligning it so the text from the button would show up to the right of the button. that's not going to work with firefox vs chrome. the spacing is different